core/src/main/java/gg/solarmc/loader/Transaction.java

```diff
@@ -39,10 +39,10 @@
 	/**
 	 * Retrieves a property of this transaction
 	 *
-	 * @param cls the class of the property
+	 * @param propertyClass the class of the property
```

Comment:
Just a suggestion but you may want to specify that this method needs to return an instance of dslcontext, sqltransaction, etc

Reply:
Remember, the API cannot know about the implementation : )